src/transformers/generation_tf_utils.py

```diff
@@ -62,87 +67,76 @@ def generate(
         decoder_start_token_id=None,
         use_cache=None,
     ):
-        r""" Generates sequences for models with a LM head. The method currently supports greedy or penalized greedy decoding, sampling with top-k or nucleus sampling
-        and beam-search.
-
-        Adapted in part from `Facebook's XLM beam search code`_.
+        r"""
+        Generates sequences for models with a language modeling head. The method currently supports greedy decoding,
```

Comment:
we could mayb add here that this applies to all `AUTO_MODEL_FOR_CAUSAL_LM` and `AUTO_MODEL_FOR _SEQ2SEQ` -> not sure though whether this is very useful for the user

Reply:
Those are not documented so it's not really useful inside the documentation.